tests/test_data/core/core__patient.txt

```diff
@@ -1,50 +1,50 @@
-('female', datetime.date(2002, 1, 1), 22, '672', '0734762a-9db6-22fc-b595-c3e472bb2a9a', 'Patient/0734762a-9db6-22fc-b595-c3e472bb2a9a', 'white', 'not hispanic or latino')
```

Comment:
This was a case where all the patient's birthdates were getting set to january first! Yeesh!